swagger.config.ts

```diff
@@ -905,10 +905,53 @@ const options = {
           },
         },
       },
+      "/api/chat/post/add": {
+        post: {
+          summary: "Send public message",
+          tags: ["Chat"],
+          security: [{ bearerAuth: [] }],
+          
+          requestBody: {
+            content: {
+              "application/json": {
+                schema: {
```

Comment:
Also add the swagger documentation of the Get request route to view all messages